from predictor import logger , CustomException

logger.info('hello')